src/resources/resources.css

```diff
@@ -81,9 +82,77 @@
 	display: block;
 }
 
+/* TODO: Add pagination css */
 .pagination-container {
 	margin: auto;
+}
+
+/* List Buttons To Render Items from Database */
+.resource-list-item {
 	display: flex;
+	flex-direction: column;
+	padding-left: 0rem;
 }
 
-/* TODO: Add pagination css */
+.resource-list-title {
+	position: relative;
+	text-align: left;
+	display: block;
+	padding: 2rem 0.75rem 2rem 0.75rem;
+	color: white;
+	background-color: transparent;
+	border: 0;
+	border-bottom: 2px solid #ccc;
+	font: 1.3rem serif;
+	transition: all 300ms;
+}
+
+.resource-list-title:hover {
+	color: #ff9617;
+	z-index: 1;
+	text-decoration: none;
+}
+
+.resource-list-title-active {
```

Comment:
For active class, I recommend change `.resources-list-title-active` into `.resources-list-title.active`. `.active` sticks at the end is the extension of `resources-list-title`. In html you just to add className="resources-list-tile active"